stac_pydantic/api/extensions/query.py

```diff
@@ -26,16 +32,27 @@ class Operator(str, AutoValueEnum):
     """
 
     eq = auto()
-    ne = auto()
+    ne = auto()  # deprecated
+    neq = auto()
     lt = auto()
-    le = auto()
+    le = auto()  # deprecated
+    lte = auto()
     gt = auto()
-    ge = auto()
+    ge = auto()  # deprecated
+    gte = auto()
     startsWith = auto()
     endsWith = auto()
     contains = auto()
 
     @DynamicClassAttribute
     def operator(self) -> Callable[[Any, Any], bool]:
         """Return python operator"""
+        if self._value_ in ["ne", "ge", "le"]:
+            newvalue = self._value_.replace("e", "te")
+            warnings.warn(
+                f"`{self._value_}` is deprecated, please use `{newvalue}`",
+                DeprecationWarning,
+                stacklevel=3,
+            )
```

Comment:
It's quite hard to add deprecation warning on Enums 😓 so for now I think adding a warning on the `operator` method is a least we can do